create.py

```diff
@@ -1,7 +1,16 @@
 from cal import app
-from cal.schema import db
+from cal.schema import db, User
 from cal.fb import update_fb_events
+import yaml
```

Comment:
I think that third-party library imports should go above local app-specific imports.